api/room.go

```diff
@@ -77,6 +77,33 @@ func (r *Room) Handle(c *websocket.Conn, l *LoginReq) error {
 			}
 			r.mu.Unlock()
 			r.broadcast(marshalChatBatch(cm.ToItem()))
+		case "kick_user":
+			msg, err := unmarshalAs[KickMessage](raw)
+			if err != nil {
+				return fmt.Errorf("failed to unmarshal kick message: %s", err)
+			}
+			if msg.User == l.User {
+				r.broadcast(marshalChatBatch(ChatBatchItem{
+					User:    "SYSTEM",
+					Text:    fmt.Sprintf("%s was trieed to kick himself", msg.User),
+					Created: time.Now().UTC().Format("2006-01-02 15:04:05"),
+				}))
+			} else {
```

Comment:
let's add `continue` such that you don't need an else branch:

if ... {A} else {B}

turns into
if ... { 
    A 
    continue 
}
B